### PR TITLE
Tests do not merge 250 beef rc 2

### PR DESCRIPTION
**Pull request by kuba-4chain**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/BuxOrg/bux/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/BuxOrg/bux/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: model_bump.go [hunk 1]
```diff
@@ -35,37 +36,58 @@ type BUMPLeaf struct {
 }
```
Comment (arkadiuszos4chain) [inline model_bump.go:36]: Please add `omitempty` to Hash field

File: model_bump_test.go [hunk 2]
```diff
@@ -70,260 +99,512 @@ func TestBUMPModel_CalculateBUMP(t *testing.T) {
...
+		bump, err := CalculateMergedBUMP(bumps)
 
 		// then
 		assert.NoError(t, err)
 		assert.Equal(t, expectedBUMP, bump)
 	})
 
-	t.Run("Slice of Merkle Proofs", func(t *testing.T) {
+	// Cannot test unless we get real BUMPs - Merkle Proofs don't match in these examples
+	// For test on real data look at the last test in this file.
```
Comment (arkadiuszos4chain) [inline model_bump_test.go:110]: If we have such tests already shouldn't we just remove this commented one?

File: model_bump.go (hunk 1 above)
Comment (kuba-4chain) [reply model_bump.go:36]: Good point. Done

File: model_bump_test.go (hunk 2 above)
Comment (kuba-4chain) [reply model_bump_test.go]: You're right, done

**Pull request closed without merging**